internal/examples/config/client.yaml

```diff
@@ -0,0 +1,17 @@
+name: myservice-client
```

Comment:
I don't think we actually need `name` in config - if I remember right, we regretted this in rpcinit.